test_results fail on OSX

test_results fail on osx, with

```
_________________________________________________________________________________________ TestResults.testResults __________________________________________________________________________________________

args = (<test_results.TestResults testMethod=testResults>,)

    def wrapped_test_method(*args):
        addExpectedRequest(method, fullurl, **kwargs)
>       test_method(*args)

/Users/jayvdb/inv/osc/tests/common.py:160:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
/Users/jayvdb/inv/osc/tests/test_results.py:49: in testResults
    out = self._run_osc('results', '--xml', 'testproject', 'python-MarkupSafe')
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <test_results.TestResults testMethod=testResults>, args = ('results', '--xml', 'testproject', 'python-MarkupSafe')

    def _run_osc(self, *args):
        """Runs osc, returning captured STDOUT as a string."""
        try:
>           os.getcwd()
E           FileNotFoundError: [Errno 2] No such file or directory

/Users/jayvdb/inv/osc/tests/test_results.py:23: FileNotFoundError
```

The problem only occurs after running test_repairwc.py before test_results.py, i.e. `pytest -v test_repairwc.py test_results.py` fails but `pytest -v test_request.py test_results.py` doesnt.

--- a/tests/common.py
+++ b/tests/common.py
@@ -182,6 +182,7 @@ def addExpectedRequest(method, url, **kwargs):
 
 class OscTestCase(unittest.TestCase):
     def setUp(self, copytree=True):
+        os.chdir(os.path.dirname(__file__))
         oscrc = os.path.join(self._get_fixtures_dir(), 'oscrc')
         osc.core.conf.get_config(override_conffile=oscrc,
                                  override_no_keyring=True, override_no_gnome_keyring=True)

